test/e2e/test-apps/other/window-titles/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "window-titles",
+  "version": "1.0.0",
+  "main": "src/main.js",
```

Comment:
What could be kinda cool is that we use https://docs.npmjs.com/cli/v9/configuring-npm/package-json#homepage to link to the issue that this is a regression test for, but just a nit.